plugins/superio/fu-superio-it55-device.c

```diff
@@ -47,6 +48,7 @@ struct _FuEcIt55Device {
 	FuSuperioDevice parent_instance;
 	gchar *prj_name;
 	AutoloadAction autoload_action;
+	gboolean me_locked;
```

Comment:
I think you need to print this in the to_string() vfunc -- e.g. https://github.com/fwupd/fwupd/blob/main/plugins/cfu/fu-cfu-device.c#L27